support-models/src/main/scala/com/gu/support/workers/CheckoutFailureReasons.scala

```diff
@@ -51,6 +54,18 @@ object CheckoutFailureReasons {
     override def asString = "production_test_account_mismatch"
   }
 
+  case object AmazonPayTryAnotherCard extends CheckoutFailureReason {
```

Comment:
these failure reasons are copied from the old logic in payment-api (see AmazonPayApiError.scala)